KMB/kmb_massload.py

```diff
@@ -225,12 +224,20 @@ def process_license(entry):
 
 def kmb_wrapper(idno):
     """Get partially processed dataobject for a given kmb id."""
-    A = {'ID': idno}
-    fil = urllib2.urlopen('http://kulturarvsdata.se/raa/kmb/' + idno)
-    dom = parse(fil)
-    fil.close()
-    del fil
-    return parser(dom, A)
+    A = {'ID': idno, 'problem': []}
+    url = 'http://kulturarvsdata.se/raa/kmb/{0}'.format(idno)
+    try:
+        fil = urllib2.urlopen(url)
```

Comment:
I likely used it to avoid using the word _file_ which is a builtin in python. For the same reason I often end up using _typ_ instead of _type_.

That said I'm changing it to `f` which is more common.